source/_integrations/gdacs.markdown

```diff
@@ -0,0 +1,137 @@
+---
+title: Global Disaster Alert and Coordination System (GDACS)
+description: Instructions on how to integrate the Global Disaster Alert and Coordination System (GDACS) feed into Home Assistant.
+logo: gdacs.jpg
+ha_category:
+  - Geolocation
+ha_iot_class: Cloud Polling
+ha_release: 0.106
+ha_config_flow: true
+ha_quality_scale: silver
+ha_codeowners:
+  - '@exxamalte'
+---
+
+The `gdacs` integration lets you use a GeoRSS feed provided by 
+[GDACS](https://www.gdacs.org/) with information 
+about major droughts, earthquakes, floods, tropical cyclones, tsunamis and 
+volcanic activities worldwide.
+It retrieves alerts from a feed and shows information of those alerts filtered 
+by distance to Home Assistant's location.
+
+Entities are generated, updated and removed automatically with each update 
+from the feed. Each entity defines latitude and longitude and will be shown 
+on the default map automatically, or on a map card by defining the source 
+`gdacs`. The distance is available as the state of each entity, and 
+converted to the unit (kilometers or miles) configured in Home Assistant.
+
+<p class='img'>
+  <img src='{{site_root}}/images/screenshots/gdacs-alerts-feed-map.png' />
+</p>
+
+The data is updated every 5 minutes.
+
+<div class='note'>
+
+The material used by this integration is provided by the [Global Disaster 
+Alert and Coordination System (GDACS)](https://www.gdacs.org/) - a cooperation 
+framework between the United Nations and the European Commission - under the 
+[Creative Commons Attribution 4.0 International (CC BY 4.0) licence](http://creativecommons.org/licenses/by/4.0/).
+It has only been modified for the purpose of presenting the material in Home Assistant.
+Please refer to the [creator's disclaimer and terms of use notice](https://www.gdacs.org/About/termofuse.aspx) for more information.
+
+</div>
+
+## Configuration
+
+To integrate the GDACS feed use the "Integrations" feature 
+in the GUI, you find it under Configurations - Integrations, or add the 
+following line to your `configuration.yaml`.
+
+```yaml
+# Example configuration.yaml entry
+gdacs:
+```
+
+{% configuration %}
+categories:
+  description: The categories of alerts to be included. Valid categories are 'Drought', 'Earthquake', 'Flood', 'Tropical Cyclone', 'Tsunami', 'Volcano'.
+  required: false
+  type: list
+  default: None. Any alert regardless of its category will be included.
```

Comment:
If there is no default, please remove it.
```suggestion
  default: None. Any alert regardless of its category will be included.
```

Reply:
OK